assets/ts/auth/access-token.ts

```diff
@@ -2,27 +2,32 @@ import { navigateTo } from "#imports";
 
 export function tradeCodeForToken(code: string, redirectUrl: URL) {
     // login() only returns a code challenge. This function turns the code challenge into an auth-token
+
+    const codeVerifier = localStorage.getItem("code-verifier");
+    if (codeVerifier === null) {
+        throw new Error("missing code-verifier for fetching access token");
+    }
     const grant = {
         grant_type: "authorization_code",
-        code: code,
+        code,
```

Comment:
kannst du da `code: code` schreiben, ich weiß ist nicht nötig, aber ist hier "fehl" am platz. (wenn du es unbedingt willst, dann `code` bitte an den anfang schieben

Reply:
bei code: code hat eslint rumgemeckert. Wieso findest dun es fehl am Platz, wenn code an zweiter Stelle steht? Ich finde es schon sinnvoll, dass in dem Grant der grant_type an erster stelle steht...